subsys/bluetooth/mesh/shell/shell_lc_cli.c

```diff
@@ -0,0 +1,339 @@
+/*
```

Comment:
Minor: could the file be renamed to `shell_ctrl_cli.c` so that it follows the same naming as other light controller files?